rules/linux/lnx_install_root_certificate.yml

```diff
@@ -0,0 +1,20 @@
+title: Install Root Certificate
+id: 78a80655-a51e-4669-bc6b-e9d206a462ee
+description: Detects installed new certificate
+references:
+    - https://attack.mitre.org/techniques/T1553/004/
+author: Ömer Günal
+date: 2020/10/05
+tags:
+    - attack.defense_evasion
+level: low
+logsource:
+    product: linux
+detection:
+    keyword:
+        - 'mv * /usr/local/share/ca-certificates'
+    keyword2:
+        - '*update-ca-certificates*'
```

Comment:
could you please use `contains` [value modifier](https://github.com/Neo23x0/sigma/wiki/Rule-Creation-Guide#value-modifiers) here as well?